projects/schematics/src/shared/lib-configs/organization-schematics-config.ts

```diff
@@ -107,3 +111,45 @@ export const ORGANIZATION_ORDER_APPROVAL_SCHEMATICS_CONFIG: SchematicConfig = {
   },
   dependencyFeatures: [USER_PROFILE_FEATURE_NAME, ORDER_FEATURE_NAME],
 };
+
+export const ORGANIZATION_ACCOUNT_SUMMARY_MODULE_NAME =
+  'organizationAccountSummary';
+export const ACCOUNT_SUMMARY_MODULE = 'AccountSummaryModule';
+export const ORGANIZATION_ACCOUNT_SUMMARY_FEATURE_NAME_CONSTANT =
+  'ORGANIZATION_ACCOUNT_SUMMARY_FEATURE';
+export const ACCOUNT_SUMMARY_TRANSLATIONS = 'accountSummaryTranslations';
+export const ACCOUNT_SUMMARY_TRANSLATION_CHUNKS_CONFIG =
+  'accountSummaryTranslationChunksConfig';
+
+export const ORGANIZATION_ACCOUNT_SUMMARY_SCHEMATICS_CONFIG: SchematicConfig = {
```

Comment:
if using wrapper module, then you can take a look of the `checkout-schematics-config.ts`

Reply:
i will have to look into it

Reply:
I am not using wrapper module, using feature module.